env/.env.prod.hydra

```diff
@@ -0,0 +1,13 @@
+DATABASE_PASSWORD="hydraDock3r!"
+DATABASE_HOST=db 
+RAILS_MAX_THREADS=32  
+FEDORA_DEV_URL=http://fcrepo:8080/rest/
+FEDORA_PASSWORD=fedoraAdmin
+SOLR_DEV_URL=http://solr:8983/solr/hydra_dev/
+TERM=xterm
+REDIS_HOST=redis
+REDIS_PORT=6379 
+REDIS_URL_SIDEKIQ=redis://redis:6379/12
+MEMCACHED_HOST=memcached
+RAILS_ENV=development
+RACK_ENV=development
```

Comment:
The filename implies that this is meant for a production environment. Given that: 

1. Why have these been added? 
2. If these are prod values, are these now leaked secrets?

Reply:
1. You will need these files to seed the db
2. These were already in the repo in the env.dev.db so they aren't true production values
we can remove these before committing this MR, but they are here to get the app running.